### PR TITLE
Updated Dependencies

### PR DESCRIPTION
**Pull request by matheusvetor**
Fixed dependencies when using with rails 5.1 and carrierwave 1.2

### CODE REVIEW

File: carrierwave_backgrounder.gemspec [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-  s.require_paths = ["lib"]
+  s.files         = `git ls-files`.split('\n')
+  s.test_files    = `git ls-files -- {test,spec,features}/*`.split('\n')
+  s.executables   = `git ls-files -- bin/*`.split('\n').map{ |f| File.basename(f) }
+  s.require_paths = ['lib']
 
-  s.add_dependency "carrierwave", [">= 0.5", "< 2.0"]
-  s.add_dependency "mime-types", ["~> 2.99"]
+  s.add_dependency 'carrierwave', ['~> 1.2']
+  s.add_dependency 'mime-types', ['~> 3.1']
```
Comment (seuros) [inline carrierwave_backgrounder.gemspec]: can we remove this dependency ? This gem is not used in the gem.

Comment (matheusvetor) [reply carrierwave_backgrounder.gemspec]: 👍

File: carrierwave_backgrounder.gemspec [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
+  s.email       = ['larry@lucidbleu.com']
+  s.homepage    = 'https://github.com/lardawge/carrierwave_backgrounder'
+  s.licenses    = ['MIT']
   s.summary     = %q{Offload CarrierWave's image processing and storage to a background process using Delayed Job, Resque, Sidekiq, Qu, Queue Classic or Girl Friday}
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
-  s.require_paths = ["lib"]
+  s.files         = `git ls-files`.split('\n')
```
Comment (p8) [inline carrierwave_backgrounder.gemspec]: '\n' is not the same as "\n"

**Comment by lardawge** [comment]
Appreciate the clean up but to many changes and really some of them are just opinions on best practices... although I tend to agree with some of them, I am not a fan of this large of a change going in at one time. Maybe break it up.

**Pull request closed without merging**